frontend/dockerfile/cmd/dockerfile-frontend/hack/release

```diff
@@ -107,9 +107,9 @@ case $TYP in
     --output type=image,\"name=$publishedNames\",$pushFlag
   ;;
 "daily")
-  if [ -z $DAILY_TARGETS ]; then
-    # DAILY_TARGETS=""
-  fi
+  # if [ -z $DAILY_TARGETS ]; then
+  #   DAILY_TARGETS=""
+  # fi
```

Comment:
Can we just remove these lines

Reply:
they are useful when new features are added